layers-cassandra/src/main/scala/geotrellis/layers/cassandra/CassandraCollectionReader.scala

```diff
@@ -14,28 +14,33 @@
  * limitations under the License.
  */
 
-package geotrellis.spark.io.cassandra
+package geotrellis.layers.cassandra
 
 import geotrellis.tiling.{Boundable, KeyBounds}
-import geotrellis.spark.io._
-import geotrellis.layers.io.avro.codecs.KeyValueRecordCodec
-import geotrellis.layers.io.avro.{AvroEncoder, AvroRecordCodec}
-import geotrellis.spark.io.cassandra.conf.CassandraConfig
-import geotrellis.layers.io.index.MergeQueue
-import geotrellis.spark.util.KryoWrapper
+import geotrellis.layers._
+import geotrellis.layers.util.IOUtils
+import geotrellis.layers.avro.codecs.KeyValueRecordCodec
+import geotrellis.layers.avro.{AvroEncoder, AvroRecordCodec}
+import geotrellis.layers.cassandra.conf.CassandraConfig
+import geotrellis.layers.index.MergeQueue
+
 import org.apache.avro.Schema
+
 import com.datastax.driver.core.querybuilder.QueryBuilder
 import com.datastax.driver.core.querybuilder.QueryBuilder.{eq => eqs}
 
 import scala.collection.JavaConverters._
 import scala.reflect.ClassTag
+
 import java.math.BigInteger
 
 import geotrellis.layers.LayerId
 
 object CassandraCollectionReader {
   final val defaultThreadCount = CassandraConfig.threads.collection.readThreads
 
+  private case class SchemaWrapper(value: Option[Schema]) extends Serializable
```

Comment:
I had to find a new wrapper for the `Schema` as I couldn't use the `KryoWrapper`. I ended up making this simple case class which I believe should do the same thing.

Reply:
@jbouffard you don't need KryoWrapper for the non Spark case.

Reply:
Ah, that's right. Will do :+1:

Reply:
But in general `KryoWrapper` makes things Serializable with Kryo and your wrapper won't make it Serializable; Look how [KryoWrapper](https://github.com/locationtech/geotrellis/blob/f8af6e6d72749b8fb175a0c44cbc7d027a940686/spark/src/main/scala/geotrellis/spark/util/KryoWrapper.scala) works when you will have some time :)